test/integration-ebpf/src/tcx.rs

```diff
@@ -0,0 +1,79 @@
+#![no_std]
+#![no_main]
+
+use core::mem;
+
+use aya_ebpf::{
+    bindings::tcx_action_base::{TCX_NEXT, TCX_PASS},
+    macros::classifier,
+    programs::TcContext,
+};
+use aya_log_ebpf::info;
+use network_types::{
+    eth::{EthHdr, EtherType},
+    ip::{IpProto, Ipv4Hdr},
+    udp::UdpHdr,
+};
+
+#[no_mangle]
+static ORDER: i32 = 0;
+
+// Gives us raw pointers to a specific offset in the packet
+#[inline(always)]
+unsafe fn ptr_at<T>(ctx: &TcContext, offset: usize) -> Result<*mut T, i64> {
```

Comment:
what's the idea here, returning TCX_PASS as the error variant? at the top of the call stack you ignore the error anyway.

Reply:
It's just a way to "early exit" out of the program, I've seen this pattern in a few of examples